html/semantics/forms/autofocus/first-when-later.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>The first autofocus in the document wins, even if elements are inserted later</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#autofocusing-a-form-control:-the-autofocus-attribute">
+<link rel="author" title="Domenic Denicola" href="d@domenic.me">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<input autofocus>
+
+<script>
+"use strict";
+
+const input1 = document.querySelector("input");
+const input2 = document.createElement("input");
+input2.autofocus = true;
```

Comment:
You're not inserting the element. It still get focus?

Reply:
Well, the test is that input1 keeps the focus, so the test still passes. But yeah, this is a bug that makes the pass trivial. Fixing.